src/components/HelpScoutBeacon/HelpScoutBeacon.js

```diff
@@ -515,26 +518,16 @@ const Main = styled.main`
   display: flex;
   flex-direction: column;
   padding: ${5 * GU}px ${3 * GU}px ${3 * GU}px;
+  color: ${({ theme }) => theme.helpSurfaceContent};
 `
 
 const Heading = styled.h3`
-  font-weight: bold;
-  color: #352c47;
-  font-size: 26px;
-  line-height: 40px;
-
-  ${breakpoint(
-    'medium',
-    `
-      font-size: 20px;
-      line-height: 31px;
-    `
-  )}
+  ${textStyle('title4')};
 `
-
-const StyledSafeLink = styled(SafeLink).attrs({ target: '_blank' })`
+console.log('Link: ', Link)
+const StyledLink = styled(Link).attrs({ target: '_blank' })`
   text-decoration: none;
-  color: ${({ color }) => color};
+  color: ${({ theme }) => theme.link};
 
   &:hover,
```

Comment:
cc @bpierre I'm not sure, but I think most of our links now don't have the underline styling on hover or focus, so we may be able to remove the custom styles entirely and just use `Link`.

`Link` also uses `theme.link` by default.